### PR TITLE
add support for LTE interface

### PR DESCRIPTION
**Pull request by mastier**
Hi,
I hope you find it useful. I made a sincere rip-off from wlanif_collector.go for supporting lte interface parameters like RSSI, RSRP, RSRQ, SINR.

Example Grafana dashboard:
image

### CODE REVIEW

**Comment by lexfrei** [comment]
Great work!

@nshttpd, can you merge this PR plz?

**Comment by paran1** [comment]
This patch was just what I needed. It works perfectly, thanks!

I only monitor a single device: wAP R ac with a R11e-LTE6 modem.

**Comment by nshttpd** [comment]
@mastier .. I've realized that I'm not dead. Looks like there are conflicts now that I've started catching up on PRs. Can you pull master and rebase to see if that's an easy fix for the conflicts?

**Comment by mastier** [comment]
@nshttpd good to hear you alive :-) I've rebased my branch with the current master. In case of issues let me know.

File: config/config_test.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ func TestShouldParse(t *testing.T) {
 	assertFeature("WlanSTA", c.Features.WlanSTA, t)
 	assertFeature("WlanIF", c.Features.WlanIF, t)
 	assertFeature("Ipsec", c.Features.Ipsec, t)
+	assertFeature("Lte", c.Features.Ipsec, t)
```
Comment (nshttpd) [inline config/config_test.go:32]: Shouldn't this be `c.Features.Lte`?

**Review by nshttpd — CHANGES_REQUESTED** [review]
Just the one thing for the config test from possibly some missed copy-pasta.

File: config/config_test.go (hunk 1 above)
Comment (mastier) [reply config/config_test.go]: > Shouldn't this be `c.Features.Lte`?

Yes. My bad. Sent fix.

**Review by nshttpd — APPROVED** [review]

**Comment by nshttpd** [comment]
Sweet. Thanks @mastier. Sorry this took me so long to get around to.

**Pull request merged**